common/views/components/MetaUnit/MetaUnit.js

```diff
@@ -2,6 +2,7 @@
 import type { Node } from 'react';
 import { spacing, font } from '../../../utils/classnames';
 import NextLink from 'next/link';
+import { type WorkTagType, default as WorkTags } from '../WorkTags/WorkTags';
```

Comment:
`import WorkTags, { type WorkTagType } from '../WorkTags/WorkTags';` Works too I think.

Reply:
Ah nice – that's probably better.